### PR TITLE
Add CI step to run `hello-world` exercise as a student would

### PR DESCRIPTION
**Pull request by clechasseur**
This PR adds a CI step to run the `hello-world` exercise as a student would (and not via the whole project workspace). This allows reproduction of bugs like the one fixed by this PR: #1406

### CODE REVIEW

File: .github/workflows/pr.ci.js.yml [hunk 1]
```diff
@@ -53,3 +53,34 @@ jobs:
...
+          rm -f hello-world.ts
+          cp ./.meta/proof.ci.ts hello-world.ts
+
+      - name: Install project dependencies
+        working-directory: exercises/practice/hello-world
+        run: yarn install --no-immutable
+
+      - name: Run tests
+        working-directory: exercises/practice/hello-world
+        run: yarn test
```
Comment (ErikSchierboom) [inline .github/workflows/pr.ci.js.yml:86]: I would actually suggest to run this for _all_ exercises. But I'm fine with that being done in a follow-up PR.

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
CI is failing though

File: .github/workflows/pr.ci.js.yml (hunk 1 above)
Comment (clechasseur) [reply .github/workflows/pr.ci.js.yml:86]: That's a good point, I'll see what I can do.

**Comment by clechasseur** [comment]
> CI is failing though

I rebased the PR, which should make it pass (hopefully). I'll work on doing this for all exercises.

**Pull request merged**